extra-radio/qsstv/autobuild/patch

```diff
@@ -0,0 +1,2 @@
+abinfo 'Patching the qsstv.pro: rename qwt-qt5 to qwt after LIBS variale...'
```

Comment:
Good morning, your personal grammar nazi here:

- "The" before "qsstv.pro" is not necessary.
- Variable *